AUTHORS

```diff
@@ -35,3 +35,5 @@ Jörg Kreuzberger <j.kreuzberger@procitec.de>
 Duodu Randy <duodurandy19@gmail.com>
 
 Christian Wappler <chri.wapp@gmail.com>
+
+Nhat Nam Nguen <nhatnamnguyen.gtvthcm@gmail.com>
```

Comment:
This might seem a bit mean, but you will have to remove your name here because you do not fullfill the process requirements for open-source contribution from the company. We can do the needful steps to fix this soon. For this PR, I will have to do this on your behalf.

Reply:
ok no problem, I will by pass it